### PR TITLE
Add PhantomJS support

### LINKED ISSUE #14: PhantomJS integration?
Hello!
Great work on an awesome framework. 

I was wondering if there were plans to make a Factory equivalent for PhantomJS to run headless browser tests. 

If not, any advice on integrating that with the existing scalatestplus framework would be nice in the meanwhile. 

Cheers

### PR DESCRIPTION
**Pull request by marcospereira**
Fixes #14.

### CODE REVIEW

File: module/src/main/scala/org/scalatestplus/play/PhantomJSFactory.scala [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+   * @return an new instance of a Selenium [[PhantomJSDriver]] or a `BrowserFactory.UnavailableDriver` if a PhantomJS
+   * driver is not available on the host platform.
+   *
+   * @see [[PhantomJSDriver]]
+   */
+  def createWebDriver(capabilities: DesiredCapabilities): WebDriver = {
+    try {
+      new PhantomJSDriver(capabilities)
+    } catch {
+      case ex: Throwable => UnavailableDriver(Some(ex), Resources("cantCreatePhantomJSDriver", ex.getMessage))
```
Comment (gmethvin) [inline module/src/main/scala/org/scalatestplus/play/PhantomJSFactory.scala]: `case NonFatal(ex)` just in case we have a fatal exception here.

Comment (marcospereira) [inline module/src/main/scala/org/scalatestplus/play/PhantomJSFactory.scala]: Will do that in another PR, since this is done for all the factories.
